internal/compiler/widgets/common/slider-base.slint

```diff
@@ -3,43 +3,51 @@
 
 export component SliderBase {
     in property <bool> enabled <=> touch-area.enabled;
-    in property <float> minimum;
+    in property <float> minimum: 0;
     in property <float> maximum: 100;
     in property <Orientation> orientation;
     in property <length> ref-width;
     out property <bool> pressed <=> touch-area.enabled;
     out property <bool> has-hover <=> touch-area.has-hover;
     out property <bool> vertical: root.orientation == Orientation.vertical;
     out property <bool> has-focus <=> focus-scope.has-focus;
-    in-out property <float> value;
+    in-out property <float> value: 0;
 
     callback changed(/* value */ float);
 
     forward-focus: focus-scope;
-    
+
     touch-area := TouchArea {
         property <float> pressed-value;
+        property <bool> pointer-moved;
 
         width: 100%;
         height: 100%;
 
         pointer-event(event) => {
             if (event.button == PointerEventButton.left && event.kind == PointerEventKind.down) {
                 self.pressed-value = root.value;
+                self.pointer-moved = false;
+                return;
+            }
+
+            if (!self.pointer-moved && event.button == PointerEventButton.left && event.kind == PointerEventKind.up) {
+                 root.set-value((!root.vertical ? root.size-to-value(touch-area.mouse-x, root.width) :
+                    root.size-to-value(touch-area.mouse-y, root.height)) + root.minimum);
+                self.pointer-moved = false;
```

Comment:
I'm curious, why is `pointer-moved` set to false here?

Reply:
To cleanup the state. But now I am thinking about it, I think to set it by `down` should be enough.